pkg/capabilities/consensus/ocr3/factory.go

```diff
@@ -0,0 +1,73 @@
+package consensus
+
+import (
+	"context"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+)
+
+type config struct {
+	RequestTimeout *time.Duration
+	BatchSize      int
+
+	clock clockwork.Clock
+}
+
+type factoryService struct {
+	store *store
+	*capability
+	batchSize int
+
+	services.StateMachine
```

Comment:
Does it need to be a service? Do we expect it to run any internal goroutines?

Reply:
Probably not, but it's required to be a service by the factory interface (and it should start the capability service as well)